core/project.go

```diff
@@ -378,8 +379,9 @@ func (p *Project) getResolver(ctx context.Context, gw bkgw.Client, r *router.Rou
 			llb.AddMount(tmpMountPath, llb.Scratch(), llb.Tmpfs()),
 		)
 
-		if p.Config.SDK == "go" {
-			st.AddMount("/src", wdState, llb.Readonly) // TODO: not actually needed here, just makes go server code easier at moment
+		switch p.Config.SDK {
+		case "go", "python":
+			st.AddMount("/src", wdState, llb.Readonly)
```

Comment:
@sipsma I only added this to access `client.host()` in the commands, but otherwise not required. I know you have other plans to improve accessing the host.